sakura_core/dlg/CDlgPluginOption.cpp

```diff
@@ -159,7 +158,7 @@ void CDlgPluginOption::SetData( void )
 		}
 
 		if (cOpt->GetType() == OPTION_TYPE_BOOL) {
-			wcscpy( buf, sValue == wstring( L"0") || sValue == wstring( L"") ? BOOL_DISP_FALSE : BOOL_DISP_TRUE );
+			wcscpy( buf, sValue == wstring( L"0") || sValue == wstring() ? BOOL_DISP_FALSE : BOOL_DISP_TRUE );
```

Comment:
```suggestion
			wcscpy_s( buf, sValue == L"0"s || sValue.empty() ? BOOL_DISP_FALSE : BOOL_DISP_TRUE );
```

Reply:
@berryzplus, this not work on MSVC compiler
```cpp
sValue == L"0"s
```

```
C++ user-defined literal operator not found
```

Reply:
`L"0"s` requires `using` statement as follows.
```c++
using namespace std::string_literals;
```
I made additional commit [1c71b64](https://github.com/sakura-editor/sakura/pull/1860/commits/1c71b64ecaf73a0b35662aa59289afe5b93158ad) to fix issue.